docs/source/collect/sysctl.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Sysctl
+description: Collect kernel parameters related to networking
+---
+
+The `sysctl` collector will read kernel parameter settings from /proc/sys/net/ipv4 and /proc/sys/net/bridge on all nodes.
+This collector schedules a pod on every node using the specified image.
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `sysctl` collector accepts the following parameters:
+
+##### `namespace` (Optional)
+The namespace where the pods will be created.
+If not specified, it will assume the "current" namespace that the kubectl context is set to.
```

Comment:
s: it will assume the "current" namespace that the kubectl context is set to \ the namespace that is currently set for the kubectl context is used